src/data_generator/jpeg_generator.py

```diff
@@ -46,7 +46,7 @@ def generate(self):
                 dim1, dim2 = [max(int(d), 0) for d in random.normal(dim, dim_stdev, 2)]
             else:
                 dim1 = dim2 = dim
-            records = random.randint(255, size=(dim1, dim2, 3), dtype=np.uint8)
+            records = random.randint(255, size=(dim1, dim2), dtype=np.uint8)
```

Comment:
We cant for two reasons.
- this will create a larger image size than described by record length.
- with the new data loader the dimensions of array should similar for all data formats. If the jpeg and png use 3d then the dataloader would not work generically.

In terms of I/O I think it is the same thing whether we use 2D or 3D till number of bytes are same for both.